rktl_perception/src/ball_detection_node.cpp

```diff
@@ -39,11 +40,6 @@ void BallDetection::BallCallback(const sensor_msgs::ImageConstPtr& msg, const se
         cv_bridge::CvImagePtr cv_ptr = cv_bridge::toCvCopy(msg, "bgr8");
         cv::Mat current_frame = cv_ptr->image;
         cv::Mat frame_HSV, frame_threshold;
-        
-        #if false /* shouldn't be necessary on the new computers */
-        /* resize image for performance */
-        cv::resize(current_frame, current_frame, cv::Size(), 0.5, 0.5);
-        #endif
```

Comment:
lmao. This is like commented out code on steroids.

Reply:
there is a reason it's gone